### PR TITLE
feat: edited descriptions for the Shears, Ceremonial Dagger and renamed Purple Duck Option

### PR DESCRIPTION
**Pull request by dribbleondo**
feat: new descriptions for three items: the Shears and Branson MD-2 Microphone, while the Burial Dagger gets a new name and description; Janus' Burial Dagger. the name is specifically a toggleable option that can be disabled. 

these changes are a part of Update 1.2.47 of [The Great Big Localisation Revamp](https://www.nexusmods.com/hitman3/mods/549) already, these have been backported here with the changes largely intact.

### CODE REVIEW

**Comment by dribbleondo** [comment]
Note to self: Github hates long titles in PR's.

**Comment by JojjeE** [comment]
Could you post the original strings for comparison's sake?

**Comment by dribbleondo** [comment]
> Could you post the original strings for comparison's sake?

Original:
**Burial Dagger**: Janus' Burial Dagger
**Shears**: Olde garden shears
**Branson MD-2 Microphone**: A vintage Branson MD-2 condenser microphone. This rare model was removed from the market in the 1960s, after causing a number of electric shock fatalities. Apparently, a production defect makes the MD-2 short-circuit at high voltages.

New:
**Janus' Burial Dagger**: A ceremonial dagger used as a part of The Ark Society's tribute to Janus.
**Shears**: A pair of old garden shears used for trimming bushes
**Branson MD-2 Microphone**: A vintage Branson MD-2 condenser microphone. This microphone was removed from the market in the 1960's due to a production defect that makes the MD-2 short-circuit at high voltages.

**Comment by Ocean-Minnow** [comment]
This seems to be more a matter of taste than a bug fix. I'm hesitant to approve something like this as it feels like it could lead to more very subjective changes in the future. Especially the Branson's new description which, whether the original is awkward or not, feels like a run-on sentence to me now.

**Comment by dribbleondo** [comment]
It being a run-on sentence is a valid point, I'll re-add that back in, just deleting the awkward comma.

Part of the reason I made the name change for the Burial Dagger an option is because it's subjective enough that it'll cause problems for those who are used to the old name (you know, for all those Hardcore Sgail speedrunners!)

As for Shears....I really don't see a downside. The description is, at best, broken, and at worst, a placeholder.

**Comment by Ocean-Minnow** [comment]
- "1960's" is incorrect, it should be "1960s" as in the original since it isn't possessive.
- I think that the Burial Dagger's description being very basic is more of a style change. The original description isn't necessarily *good*, but I don't think it's within the scope of this mod to decide what it "should" be.
- The same as above goes for the shears, aside from the typo in "olde" which should probably be fixed.

**Comment by dribbleondo** [comment]
I have removed the altered description for the Burial Dagger; i've kept the name change for the Burial Dagger as an option, though this may lead to the amusing situation of the name and description being identical =P

Also, yeah, I did change the Purple Streak option names, mostly to make them a smidge more obvious to the end-user.

**Comment by Ocean-Minnow** [comment]
Now you've written "shearss" :p

As for the burial dagger's name... this is where it gets a bit awkward. Despite the default description, it doesn't seem to always be *Janus'* burial dagger. You can find two in New York (one in the vault, one in a deposit box) and with it being unlockable now, that means you can potentially have a total of three daggers at once.

I'm not entirely sure how to handle this, since just changing it to something like "A burial dagger" would be even worse than the default description, but writing a whole new one seems a bit like overreach. Right now I'm leaning towards "just leave it be" but I'm open to suggestions.

**Comment by dribbleondo** [comment]
image
Oh joy.

**Comment by Ocean-Minnow** [comment]
Any suggestions on how to handle the burial dagger's name, @dribbleondo? I wouldn't be opposed to including it if maybe it were set to off by default. I wouldn't want it to be on by default since it does highlight this oddity in New York.

**Comment by dribbleondo** [comment]
> Any suggestions on how to handle the burial dagger's name, @dribbleondo? I wouldn't be opposed to including it if maybe it were set to off by default. I wouldn't want it to be on by default since it does highlight this oddity in New York.

I think the only solution is to genercise the description and remove the naming option; that way it makes sense everywhere.

So instead of `Burial Dagger: Janus' Burial Dagger` it could be `Burial Dagger: A ceremonial dagger.`, which i'm not fond of as things go, but it'll have to do. Thoughts?

**Comment by Notexe** [comment]
Should I convert this PR into a draft? Just so it doesn't get accidentally merged if changes are still being made to it.

**Comment by Ocean-Minnow** [comment]
Sounds reasonable enough to me!

File: manifest.json [hunk 1]
```diff
@@ -45,6 +58,16 @@
...
+		"008322D98E1CFE04": {
+			"english": {
+				"611928695": "Old garden shears."
+			}
+		},
+		"00873D29DB58C88A": {
+			"english": {
+				"594769380": "A vintage Branson MD-2 condenser microphone. This rare model was removed from the market in the 1960s after causing a number of electric shock fatalities. Apparently, a production defect makes the MD-2 short-circuit at high voltages."
+			}
+		},
```
Comment (RDIL) [inline manifest.json:70]: This feels to me like an opinionated change, -1 on adding.

Comment (Ocean-Minnow) [reply manifest.json:70]: I do have to say I agree. I think I lost track of it a bit in all the shuffling around, but the comma really does seem like a matter of style. It's not an Oxford comma, but the same principle applies.

Comment (dribbleondo) [reply manifest.json:70]: If I remember from Discord, you said it was written like as if someone was speaking, which doesn;t always translate well to text, hence the request to change it from Stark on Discord. I honestly don't mind either way.

**Comment by dribbleondo** [comment]
I've removed the Branson description considering it seems to be agreed that it's a style change. I have re-added the Burial Dagger description to "A ceremonial dagger." which is more generic and means it works in more places.

**Comment by Notexe** [comment]
Is 2935068597 in 00873D29DB58C88A.LOCR though? Since 00873D29DB58C88A is for tiger specific items.

**Comment by dribbleondo** [comment]
005DE6FF4B418AF8 is the correct one, 00873D29DB58C88A is wrong, good catch!

008322D98E1CFE04 is Items_s3:
image

**Comment by Notexe** [comment]
> 008322D98E1CFE04 is Items_s3:

Oh oops I had GK loading the H2016 files still, my bad lmao.

**Pull request merged**